File: Data/HDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebThietBi.Data
{
    public class HDB
    {
        private string id_HDB;
        private DateTime ngayLapHDB;
        private string id_KhachHang;
        private string id_SanPham;
        private string tenSanPham;
        private int soluong;
        private int donGia;


        public string Id_HDB { get => id_HDB; set => id_HDB = value; }
        public DateTime NgayLapHDB { get => ngayLapHDB; set => ngayLapHDB = value; }
        public string Id_KhachHang { get => id_KhachHang; set => id_KhachHang = value; }

        public string Id_SanPham { get => id_SanPham; set => id_SanPham = value; }
        public string TenSanPham { get => tenSanPham; set => tenSanPham = value; }
        public int Soluong { get => soluong; set => soluong = value; }

        public int DonGia { get => donGia; set => donGia = value; }

    }
}

File: Front-end/js/HDB.js
$(document).ready(function () {
    getHDB();
});
var customer = JSON.parse(localStorage.getItem('user'))
function getHDB() {
    $.ajax({
        url: 'https://localhost:44366//api/HDB/' + customer.ID_KhachHang,
        //+ customer.ID_KhachHang',
        method: 'GET',
        contentType: 'application\json',
        dataType: 'json',
        error: function (response) { },
        success: function (response) {
            const len = response.length;
            console.log(response)
            let table = ''
            for (var i = 0; i < len; i++) {
                let money = response[i].Soluong * response[i].DonGia
                table = table + '<tr>';
                table = table + '<td>' + response[i].TenSanPham + '</td>';
                table = table + '<td>' + response[i].NgayLapHDB + '</td>';
                table = table + '<td>' + response[i].DonGia.toLocaleString('it-IT', { style: 'currency', currency: 'VND' }); + '</td>';
                table = table + '<td>' + response[i].Soluong + '</td>';
                table = table + '<td>' + money.toLocaleString('it-IT', { style: 'currency', currency: 'VND' }); + '</td>';

                table = table + '</tr>';
            }
            $("#HDB").append(table)
        },
        fail: function (response) { }
    });

}